package.json

```diff
@@ -35,7 +36,7 @@
     "skatejs-web-components": "2.0.1"
   },
   "scripts": {
-    "test": "sk-tests",
+    "test": "node dist/ && sk-tests",
```

Comment:
Fail early with running in node as this takes less time. Node will throw non-zero here, so this should be sufficient.